### ISSUE TITLE
defaultValue for argument of InputObjectType

### ISSUE THREAD

**Issue by vladar** [webonyx/graphql-php#12]
Currently Introspection for `__InputValue.defaultValue` uses `json_encode`, which works for simple input values, but fails when input type is a complex `InputObjectType`.

Currently introspection produces following value: 
```
[{"selectId":"value","selectedOptions":["a", "b", "c"]}]
```

While GraphQL JS (and Relay) expect:
```
[{selectId: "value", selectedOptions: ["a", "b", "c"]}]
```

Need to implement custom `astFromValue` helper and use `Printer` instead of `json_encode`.

**Comment by ozziest** [comment]
:+1:

**Comment by vladar** [comment]
This should be available with 0.7.0 release.
